Sources/Plasma/FeatureLib/pfJournalBook/pfJournalBook.cpp

```diff
@@ -1672,6 +1876,7 @@ bool    pfJournalBook::ICompileSource(const ST::string& source, const plLocation
     IFreeSource();
 
     pfEsHTMLChunk *chunk, *lastParChunk = new pfEsHTMLChunk(ST::string());
+    pfEsHTMLChunk *lastLinkChunk = nullptr;
```

Comment:
```suggestion
    pfEsHTMLChunk* lastLinkChunk = nullptr;
```

Reply:
That's what our code style says, strictly speaking, but we're also supposed to match the style of the immediately surrounding code. It's a tough call sometimes :(

Reply:
I run into that conflict a lot.